### PR TITLE
ES6 code for map, filter and reduce function

### PR DESCRIPTION
**Pull request by melwinalm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -671,6 +671,12 @@ const doubledNumbers = numbers.map(doubleN);
 console.log(doubledNumbers); // [0, 2, 4, 6, 8, 10, 12]
 ```
 
+In ES6
```
Comment (mbeaudru) [inline README.md]: Replace with:

> **Note :** You will frequently encounter this method used in combination with [*arrow functions*](https://github.com/mbeaudru/modern-js-cheatsheet#-arrow-function)
> ```js
> const doubledNumbers = numbers.map(n => n * 2);
> console.log(doubledNumbers); // [0, 2, 4, 6, 8, 10, 12]
> ```

File: README.md [hunk 2]
```diff
@@ -684,6 +690,13 @@ const evenNumbers = numbers.filter(function(n) {
 console.log(evenNumbers); // [0, 2, 4, 6]
 ```
 
+In ES6
```
Comment (mbeaudru) [inline README.md]: Same

File: README.md [hunk 3]
```diff
@@ -701,6 +714,13 @@ const sum = numbers.reduce(
 console.log(sum) //21
 ```
 
+In ES6
```
Comment (mbeaudru) [inline README.md]: Same

File: README.md (hunk 1 above)
Comment (melwinalm) [reply README.md]: I have made the necessary changes as mentioned above

File: README.md (hunk 2 above)
Comment (melwinalm) [reply README.md]: I have made the necessary changes as mentioned above

File: README.md (hunk 3 above)
Comment (melwinalm) [reply README.md]: I have made the necessary changes as mentioned above

**Comment by melwinalm** [comment]
Hi @mbeaudru 
I have made the necessary changes as mentioned above

**Pull request merged**